.devops/nix/package.nix

```diff
@@ -213,10 +213,16 @@ effectiveStdenv.mkDerivation (
         ;
 
       shell = mkShell {
+        NIX_LD_LIBRARY_PATH = lib.makeLibraryPath [
```

Comment:
Maybe add a comment about this variable being specific to `nix-ld`

Reply:
I copied this pattern from an example, and since I don't have `nix-ld` installed on my system, it seems like it's not necessary for the "fix", at least in my case.

so I could just float the definition up into a `let`.

Reply:
> it seems like it's not necessary for the "fix"

Sure, just let's not spread unmotivated examples for more people to copy, but spend a moment to figure out where this came from